crates/aptos-crypto/Cargo.toml

```diff
@@ -35,15 +38,14 @@ serde = { workspace = true }
 serde-name = { workspace = true }
 serde_bytes = { workspace = true }
 sha2 = { workspace = true }
+sha2_0_10_6 = { workspace = true }
```

Comment:
Why do we need this dependency?

Reply:
IIRC the one above does not work with arkworks hash-to-curve implementation.